cmd/config.go

```diff
@@ -65,7 +65,8 @@ type messagingConfig struct {
 }
 
 type tariffConfig struct {
-	Grid typedConfig
+	Grid   typedConfig
+	Feedin typedConfig
```

Comment:
FeedIn

Was ist hier gemeint? Export?

Reply:
Habs jetzt konsequent überall in `feedIn` umbenannt. Wir können auch `export` nehmen. Da bin ich inzwischen leidenschaftslos. Glaube beides ist verständlich.